cmds/exp/rush/parse.go

```diff
@@ -136,6 +136,9 @@ func tok(b *bufio.Reader) (string, string) {
 }
 
 // get an ARG. It has to work.
+// TODO: rework 'what' parameter, maybe remove it?
+//
+//nolint:unparam
 func getArg(b *bufio.Reader, what string) string {
```

Comment:
OK, I just now tested this change:
```
-func getArg(b *bufio.Reader, what string) string {
+func getArg(b *bufio.Reader, _ string) string {

rminnich@sidecore:~/go/src/github.com/u-root/u-root$ golangci-lint run cmds/exp/rush/parse.go
rminnich@sidecore:~/go/src/github.com/u-root/u-root$ 
```

golangci-lint is fine with _ parameters. 

So, the fix in the PR is the wrong fix. Adding the nolint: makes things worse. 

Either remove what, which is the better solution; or make it _ string. 

I would bet that just about every nolint: in this PR could be done a better way.